### DISCUSSION TITLE
Ignore "The" in title sort or include it?

### DISCUSSION THREAD

**Discussion by Jdiesel87** [advplyr/audiobookshelf#177] [General]
General question on whether "The" should be ignored or include when sorting in title view?

I've seen it done bother ways, what is your preference?

**Comment by advplyr** [comment]
I'm not sure. I think I prefer including "the", but I can also see if someone has a very large collection that they would prefer to exclude it.

If enough people prefer this, it could be something added to server settings.

**Comment by FlamingoDaBird** [comment]
Hi I would prefer to keep "The" excluded, as mentioned by advplyr, in a large collection it becomes tedious.
I'd like for all definite articles to be excluded not just "The".  "A" and "An" and all of them :)

Or at least implement a switch for people to choose for themselves, if it's possible.

Thank you
Best regards
Flamingo Da Bird
